src/resources/help/en_US/relnotes3.2.0.html

```diff
@@ -32,7 +32,8 @@
 <p><b><font size="4">General features added in version 3.2</font></b></p>
 <ul>
   <li>Upgraded VTKm to a recent version. Support was added for zone centered variables and 3D unstructured meshes with tetrahedra, pyramid and prism cells, in addition to the existing support for hexahedra.</li>
-  <li>General Feature 2</li>
+  <li>Added <b>-py2to3</b> CLI command line option that enables limited on-the-fly Python 2 to Python 3 conversion for scripts passed via <b>-s</b> or  <b>visit.Source()</b>. See VisIt's CLI manual for more details. </li>
```

Comment:
Typos: "Added" to "Added the". There are 2 spaces after "or" and an extra space at the end of the last sentence.

Reply:
fixed these.